lib/configuration.rb

```diff
@@ -306,8 +306,6 @@ def load_configuration(location, ref, is_release)
     # do final touchups on the configuration
     configuration.aging_pull_requests_notification = true if configuration.aging_pull_requests_notification.nil?
     configuration.aging_pull_requests_numdays = 7 if configuration.aging_pull_requests_numdays.nil?
-    configuration.test_pass_limit = 99.9999 if configuration.test_pass_limit.nil?
-    configuration.test_warn_limit = 90.00 if configuration.test_warn_limit.nil?
```

Comment:
Remove the test pass/warn limits entirely.  Now it's either 100% pass = good, anything else = bad.